### PR TITLE
Added README for tests

### PR DESCRIPTION
**Pull request by sushilkm**

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+We are also making use of a pre-provisioned batch account and we deploy a batch-account pool and job on batch-account to override some quota challenges faced in our Azure subscription.
+
+## Github secrets and credentials
+
+We make use of github secrets to provide credentials and other secret information to run the pipeline.
+
+Following is the list of secrets we are currently using:
+
+- AZURE_CREDENTIALS:
+    These are the credentials used accessing azure cloud. Follow the instructions on the [documentation](https://docs.microsoft.com/en-us/azure/developer/github/connect-from-azure?tabs=azure-cli%2Clinux#use-the-azure-login-action-with-a-service-principal-secret) to create and use the credentials.
```
Comment (sjyang18) [inline test/README.md:14]: The instruction creates sp at a certain resource group and contributor role. Will this work?

File: test/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+- CI_BATCH_ACCOUNT_NAME:
+    This is the pre-provisioned batch account name.
```
Comment (sjyang18) [inline test/README.md:17]: What does this mean for external users? Do they need to provision the batch account first?

File: test/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+- CI_STORAGE_ACCOUNT:
+    This is the storage account where we have stored the configuration files and the input file.
```
Comment (sjyang18) [inline test/README.md:20]: So, we need another pre-provisioned storage account? Do we need to create a container in it? And, how do we pass the container name to test data pipeline? Inside the container, do we need to create two directories, config and raw?

File: test/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    Following is the list of configuration files stored on storage account:
+    - `custom-vision-model-v2\config\config.json`:  Config file
+    - `custom-vision-model-v2\config\custom_vision_object_detection.json`: Config file
+    - `custom-vision-model-v2\raw/sample_4326.tif`: Input file
+
+    For more information on which files need to be in storage account take a look at [README](../deploy/README.md#running-the-pipeline-custom-vision-model-v2)
+
+- AAD_GROUP_PRINCIPAL_ID:
+    This is the principal id for team group on azure to be granted `Synapse Administrator` role.
+    Keep it blank if it is of no use to you, and scripts will skip it.
```
Comment (sjyang18) [inline test/README.md:30]: If I keep it blank, will test run in github action?

Comment (sushilkm) [reply test/README.md:30]: yes

File: test/README.md (hunk 3 above)
Comment (sushilkm) [reply test/README.md:20]: As a public user yes. the filenames have been listed as full container path as required.
This is the first cut. we can update it better with a script or more instructions what to copy and where to copy.

File: test/README.md (hunk 2 above)
Comment (sushilkm) [reply test/README.md:17]: As of now yes. We would update the test script in the next iteration that if this secret is not provided then we would create the batch account

File: test/README.md (hunk 1 above)
Comment (sushilkm) [reply test/README.md:14]: OK, I would update the instructions, it should be the subscription level only.

**Pull request closed without merging**